### PR TITLE
Fix python version in README

### LINKED ISSUE #4: Conda install only works with python=3.7
The readme says python=3.6.. 

I will fix this asap.

### PR DESCRIPTION
**Pull request by ahendriksen**
Fixes #4

### CODE REVIEW

**Comment by ahendriksen** [comment]
@jwbuurlage  @wjp : Can somebody give Rien, Adriaan, and me commit rights? Right now I cannot accept this pull request.

**Comment by jwbuurlage** [comment]
I have made you an admin, I assume Alex is okay with that.

Op vr 7 jun. 2019 om 17:59 schreef Allard Hendriksen <
notifications@github.com>:

File: README.md [hunk 1]
```diff
@@ -11,10 +11,13 @@ It takes a few steps to setup flexDATA on your machine. We recommend that the us
 
 Simply install with:
 ```
-conda create -n <your-environment> python=3.6
+conda create -n <your-environment> python=3.7
 conda install -c cicwi -c astra-toolbox/label/dev -c conda-forge -c owlas flexdata
 ```
 
+**NOTE**: currently, the latest version of flexdata is only available
+for python version 3.7.
```
Comment (jwbuurlage) [inline README.md]: **NOTE**: currently, the latest version of flexDATA is only available
for Python version 3.7.

Comment (ahendriksen) [reply README.md]: Capitalization issue?

**Comment by jwbuurlage** [comment]
Yeah, I couldn't help myself

**Comment by ahendriksen** [comment]
Thanks. Fixed.

**Pull request merged**